### ISSUE TITLE
Keybindings Not Working

### ISSUE THREAD

**Issue by Odysseus-USA** [CBATeam/CBA_A3#1476]
Most Current Version Of ARMA And CBA (07/12/2021) (CBA_A3 v3.15.4.210701)

Description:
Using a mod with the CBA keybindings configs and has been working for a long time, now, with the latest update, they do not.  You also can no longer see the config via ARMA's menu (ESC -> Controller -> Configure Addons)

Many mods I have like CUP,VCOM, my mod, and many others now no longer have keybindings nor can they be seen/modified via ARMA's menu.

When I put back the previous version of CBA, all is ok.

CBA_A3 v3.15.3.210310 = Working (03/10/2021)

CBA_A3 v3.15.4.210701 = Fails (Current as of 07/12/2021)

Steps to reproduce:
Use a mod with the keybinding commands such as CUP or VCOM.

Expected behavior:
To restore keybindings as before

Where did the issue occur?
SP

Log Files:
Cannot use PasteBin as I am not a paying member (RPT file too big), but an example from old version and new version of CBA:

Old Version (Working) CBA_A3 v3.15.3.210310:
19:35:20 "HelpMe Mod v29.32 Loaded!  (c)2021 By Super Chicken Productions Atlanta Georgia USA"
19:35:20 "VCOM: Loaded CBA settings"
19:35:20 "VCOM: Loaded CBA settings"
19:35:23 Loading movesType CfgMovesRabbit_F

New Version (Failing) CBA_A3 v3.15.4.210701:
19:10:39 "HelpMe Mod v29.32 Loaded!  (c)2021 By Super Chicken Productions Atlanta Georgia USA"
19:10:39 Error in expression <e"];
VCM_ActivateAI = _value;
} 
] call CBA_Settings_fnc_init;

[
"VCM_USECBASE>
19:10:39   Error position: <CBA_Settings_fnc_init;

[
"VCM_USECBASE>
19:10:39   Error Undefined variable in expression: cba_settings_fnc_init
19:10:39 File vcomai\Vcom\Functions\VCM_Functions\fn_CBASettings.sqf..., line 18
19:10:39 "VCOM: Loaded CBA settings"

**Additional context:**
This is a reoccurring issue at times with CBA releases that has been an issue for years.  New update, keys quit working.

**Screenshots:**
n/a

Yall do a good job and thanks for your work!

**Comment by commy2** [comment]
Post the full rpt file. Cannot help with fragments. Seems to be a local issue for you though.

**Comment by PabstMirror** [comment]
We just released 3.15.5 that should fix these issues, please let us know if there are any problems still

**Comment by Odysseus-USA** [comment]
Problem solved.

A campaign dev (for whatever reason) put you last version of CBA files inside his mod.  When the new update came out, instant conflict!


**Leave mod in:  No keybindings and 'Configure Addons' ghosted out**

**Remove mod:  EVERYTHING works**

For those who might have this issue, examine your RPT file and see if any mod is loading in 'extra' CBA files, this is how I found this out.

Mod In Question:

Name:  SP Campaign FRONT
Author:  CERBER
Link:  https://steamcommunity.com/sharedfiles/filedetails/?id=1941043158

This issue can now be closed as I have found the issue.  I will let yall close this issue out when yall are ready!  :)


As a note for you/others, _PasteBin_ has a very small limit for free accounts, which I have.  This is why I could not put in the RPT file.


Thank yall for your help and the mod itself!

:)
